### PR TITLE
update kn version

### PR DESCRIPTION
**Pull request by sudhirverma**
Fix: #207

### CODE REVIEW

**Review by lstocchi — APPROVED** [review]
LGTM

**Comment by mohitsuman** [comment]
@sudhirverma Include `kn-darwin-arm64` version, for M1 user support.

File: src/cli/cli-config.ts [hunk 1]
```diff
@@ -142,6 +149,15 @@ export class CmdCliConfig {
    */
   static async detectOrDownload(cmd: string): Promise<string> {
     try {
+      if (!macUser.get('m1MacUser') && !macUser.get('notM1MacUser') && Platform.OS === 'darwin') {
```
Comment (lstocchi) [inline src/cli/cli-config.ts:152]: I don't have a mac so i cannot test this thing at all but by looking at the code it's seems that M1 is arm64. Why don't you just do `if (process.arch === 'arm64' and darwin) download the cli arm64 `?

**Review by lstocchi — APPROVED** [review]
LGTM. Cannot test on Mac but it should work

**Comment by sudhirverma** [comment]
@mohitsuman it works with Mac M1.

**Review by mohitsuman — APPROVED** [review]

**Pull request merged**